### PR TITLE
Add MD Health Check

### PR DESCRIPTION
**Pull request by mbusk88**
## References

* #556

### CODE REVIEW

File: source/Energinet.DataHub.PostOffice.EntryPoint.MarketOperator/Energinet.DataHub.PostOffice.EntryPoint.MarketOperator.csproj [hunk 1]
```diff
@@ -33,10 +34,6 @@ limitations under the License.
     <None Update="host.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Update="local.settings.json">
```
Comment (defectiveAi) [inline source/Energinet.DataHub.PostOffice.EntryPoint.MarketOperator/Energinet.DataHub.PostOffice.EntryPoint.MarketOperator.csproj:36]: Why this change?

**Review by defectiveAi — APPROVED** [review]

**Review by FirestarJes — APPROVED** [review]
